filebeat/module/apache2/access/test/test.log-expected.json

```diff
@@ -1,77 +1,30 @@
 [
     {
         "@timestamp": "2016-12-26T14:16:29.000Z", 
-        "apache2.access.body_sent.bytes": "209", 
-        "apache2.access.http_version": "1.1", 
-        "apache2.access.method": "GET", 
-        "apache2.access.remote_ip": "::1", 
-        "apache2.access.response_code": "404", 
-        "apache2.access.url": "/favicon.ico", 
-        "apache2.access.user_name": "-", 
+        "file.path": "/Users/ruflin/Dev/gopath/src/github.com/elastic/beats/filebeat/module/apache2/access/test/test.log", 
         "fileset.module": "apache2", 
         "fileset.name": "access", 
+        "http.request.method": "GET", 
+        "http.response.body_sent.bytes": "209", 
+        "http.response.status_code": "404", 
+        "http.version": "1.1", 
```

Comment:
Same here, not in ECS but still under http.

Reply:
`http.version` is in ECS already :-)